django/contrib/auth/models.py

```diff
@@ -361,6 +360,13 @@ def email_user(self, subject, message, from_email=None, **kwargs):
         """
         send_mail(subject, message, from_email, [self.email], **kwargs)
 
+    def save(self, *args, **kwargs):
+        if self.email:
+            self.email = UserManager.normalize_email(self.email)
```

Comment:
Thanks for your suggestion. It looks better to me, will fix this in the next commit